lib/features/children/overview/widgets/profile_overview_tile.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app/app/routes/routes.dart';
+import 'package:givt_app/core/enums/amplitude_events.dart';
+import 'package:givt_app/features/auth/cubit/auth_cubit.dart';
+import 'package:givt_app/features/children/overview/cubit/family_overview_cubit.dart';
+import 'package:givt_app/features/children/overview/models/profile.dart';
+import 'package:givt_app/utils/utils.dart';
+import 'package:go_router/go_router.dart';
+
+class ProfileOverviewTile extends StatelessWidget {
+  const ProfileOverviewTile({
+    required this.profile,
+    super.key,
+  });
+
+  final Profile profile;
+  @override
+  Widget build(BuildContext context) {
+    final user = context.read<AuthCubit>().state.user;
+    final currencySymbol = Util.getCurrencySymbol(countryCode: user.country);
+    final isGivtAccount = profile.firstName == user.firstName;
+    return Container(
+      margin: const EdgeInsets.symmetric(horizontal: 5),
+      child: OutlinedButton(
+        style: OutlinedButton.styleFrom(
+          foregroundColor: _getBorderColor(context, isGivtAccount),
+          elevation: 0,
+          padding: const EdgeInsets.symmetric(
+            horizontal: 12,
+            vertical: 16,
+          ),
+          shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(8),
+          ),
+          side: BorderSide(
+              width: 2,
+              color: _getBorderColor(
+                context,
+                isGivtAccount,
+              )),
+          backgroundColor: _getBackgroundColor(context),
+        ),
+        onPressed: () => _onTap(context, isGivtAccount),
+        child: SizedBox(
```

Comment:
why do we need SizedBox here?

Reply:
No clue actually its removed, and all works the same😅